pkg/driver/videotest/dummy.go

```diff
@@ -0,0 +1,132 @@
+// Package videotest provides dummy video driver for testing.
+package videotest
+
+import (
+	"context"
+	"image"
+	"io"
+	"math/rand"
+	"time"
+
+	"github.com/pion/mediadevices/pkg/driver"
+	"github.com/pion/mediadevices/pkg/frame"
+	"github.com/pion/mediadevices/pkg/io/video"
+	"github.com/pion/mediadevices/pkg/prop"
+)
+
+func init() {
+	driver.GetManager().Register(
+		newVideoTest(),
+		driver.Info{Label: "VideoTest", DeviceType: driver.Microphone},
```

Comment:
Oops, it should be `driver.Camera` there

Reply:
oops